.github/workflows/tests.yaml

```diff
@@ -48,11 +48,11 @@ jobs:
         run: go build
       -
         name: Run library tests
-        run: go test -race -v ./...
+        run: CGO_CXXFLAGS=-fPIE CGO_CFLAGS=-fPIE CGO_LDFLAGS=-pie go test -buildmode=pie -v ./...
```

Comment:
You need to preserve the previous values of these env vars (set line 37). Same in the Docker images.